load_balancer.tf

```diff
@@ -29,16 +29,34 @@ resource "aws_lb" "public_lb" {
 
 resource "aws_lb_target_group" "ec2_tg" {
   name     = "Terraform-EC2-TargetGroup"
-  port     = 80
-  protocol = "HTTP"
+  port     = 443
```

Comment:
This change is not necessary. The target group specifies to which port of the receiving instances (in our case the Swarm Instances) will the requested be forwarded to. In our case we want the instances to listen to http on port 80 and let the LB handle the SSL